### PR TITLE
Turn off tabindex-no-positive rule

### PR DESCRIPTION
**Pull request by pecanoro**
Discussed [here](https://github.com/Expensify/Web-Expensify/pull/22798#issuecomment-432210355) to make the linter pass in the PR

### CODE REVIEW

**Comment by fnwbr** [comment]
@bondydaa @tgolen Knowing you guys have talked about accessibility in the past, can you get down to disabling this rule?

See the link Rocio posted above for my reasoning.
See https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/tabindex-no-positive.md and the AX link there for more details on the rule.

**Comment by bondydaa** [comment]
So looking at the associated PR: https://github.com/Expensify/Web-Expensify/pull/22798/files

why are these `tabIndex`s set all to `1`?
image

`tab-index` were created a primarily used so that you could create "custom" navigation flows that broke with your HTML document layout ("tab flow").

EX:
```
<input tabIndex=2 name='input1'> 
<input tabIndex=1 name='input2'>
<input tabIndex=3 name='input3'>
```

In the above scenario if you're `tab`ing through would go to `input2`, then `input1` then `input3`.

Essentially putting the same value for `tabIndex` is "resetting" the "tab flow" to treat them as if there were no `tabIndex`s set at all (top down tab order based on HTML document layout). 

Are there other `tab-index`s set on the page that are greater than `1`? Ideally, we don't need to set a `tab-index` because this is just going to lead to confusion down the line as we then start having to add `tab-index=2` then `3` because we've now set too many custom `tab-index` values. (Think about `z-index` values and how quickly that becomes confusing trying to properly layer elements on top of one another)

The rule in the linter is because with accessibility (a11y) it's very common to set `tab-index=-1` to put a custom element into the "tab flow" so that it can be given focus when `tab`ing through.

**Comment by bondydaa** [comment]
Oh sorry and all of that is meant to say that I'm not a big fan of changing this rule as it does force us to come up with solutions that don't create custom `tab-index` flows.

Yes it's an internal tool and we've always put a11y on the back burner but if possible we should avoid doing things like this.

**Comment by pecanoro** [comment]
All of those have the same value set to 1 because each job has a different number or inputs that I don't know beforehand, I simply want to jump to the submit button after the job form. If I set all of those to 1 and then I add 2 to the submit button, it works as I intend, but I can't assign incremental numbers to the input because then I will never know what to use for the next button.

**Comment by fnwbr** [comment]
Also, if you give something a negative tabIndex, it will be at the _end_ of the focus order.
So first it would focus through everything else on the page (all the links and buttons available), and **then** it would get to the elements that you gave custom (negative) `tabIndex`s. I think this rule is making our life unnecessarily hard, we have plenty of other lower hanging fruit that we could fix if we would take accessibility as our no. 1 priority.

**Comment by bondydaa** [comment]
>  I simply want to jump to the submit button after the job form

Okay so this is done to create "power user" movements. Again I'm not crazy about messing with tab flow so instead why not make "power user" keyboard shortcuts like we have in the web app (`ctrl + s` to jump to submit button or submit expense).

Keep in mind that a11y isn't just for blind people, it could be used by people who have full sight but limited mobility in their hands/arms and use software assists that track eye movement to navigate or something else.

"it's an internal tool" in my mind isn't a great reason to justify poor accessibility especially since we do give this tool to people outside the immediate company to use and we shouldn't assume that everyone uses it the way we're coding it.

**Comment by fnwbr** [comment]
Took this 1:1 with @bondydaa - @pecanoro I'll reply in your Web-Expensify PR.
tl;dr: He's right about that accessibility should not be neglected just because something is seen as an internal tool. See more notes in the other PR, this repo is public after all.
